src/modules/UI/scenes/SendConfirmation/selectors.js

```diff
@@ -118,3 +128,21 @@ export const getSpendInfo = (state: State, newSpendInfo?: GuiMakeSpendInfo = {})
     customNetworkFee: newSpendInfo.customNetworkFee ? { ...getCustomNetworkFee(state), ...newSpendInfo.customNetworkFee } : getCustomNetworkFee(state)
   }
 }
+
+export type AuthType = 'pin' | 'none'
+export const getAuthRequired = (state: State, spendInfo: EdgeSpendInfo): AuthType => {
+  const currencyCode = spendInfo.currencyCode || spendInfo.spendTargets[0].currencyCode
+  const { nativeAmount } = spendInfo.spendTargets[0]
+  if (!currencyCode || !nativeAmount) throw new Error('Invalid Spend Request')
```

Comment:
Consider defining a new error type here or using one from the `core`

Reply:
this is the same as the other spend info error
i dont expect this to ever fire in production
it's primarily to make flow happy

should "make flow happy" errors go into the global `errors.js` as well?

or maybe we come up with a `make flow happy` convention
should we use errors or `flow_fix_me`?